consensus/src/quorum_store/utils.rs

```diff
@@ -210,10 +210,11 @@ pub struct ProofQueue {
     remaining_proofs: u64,
     remaining_local_txns: u64,
     remaining_local_proofs: u64,
+    max_txn_summary_storage: usize,
```

Comment:
Nit: u64 is more consistent with all other types in the struct

Reply:
Done.